package_manifest.yaml

```diff
@@ -486,8 +512,6 @@ foreman_nonscl_packages:
     puppet-agent-yard: {}
     rubygem-foreman_maintain: {}
     rubygem-kafo: {}
```

Comment:
The subject says this should also be converted. Waiting for the 4.1.0 release?

Reply:
It was meant to say kafo_wizards. I am doing the dependencies first since kafo repoclosure will fail.

Reply:
I don't see why including Kafo wouldn't work.